mycity/mycity/mycity_data_model.py

```diff
@@ -0,0 +1,158 @@
+class MyCityDataModel:
+    """
+    Represents a request from a voice platform.
```

Comment:
This also currently represents a response as well. Maybe in the future we want to break these into two objects (request and response), but lets get this in first.

Reply:
Agreed.